cardboard/settings.py

```diff
@@ -146,14 +146,19 @@
 
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/2.2/howto/static-files/
+# https://whitenoise.readthedocs.io/en/latest/django.html
 
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
 STATIC_URL = "/static/"
 STATICFILES_DIRS = [
     os.path.join(BASE_DIR, "cardboard/static"),
     os.path.join(BASE_DIR, "hunts/static"),
 ]
-STATICFILES_STORAGE = "whitenoise.django.CompressedManifestStaticFilesStorage"
+STORAGE = {
```

Comment:
This is a typo, this setting is named `STORAGES` https://docs.djangoproject.com/en/5.0/ref/settings/#storages

But it was also added in Django 4.2. In the `pyproject.toml` we only have 4.1 specified and the package version in the `poetry.lock` is also 4.1, so you should use the 4.1 settings (we should also upgrade Django to the latest version but probably in a separate PR)

Reply:
Oh oops, you're right. I'll delete my `staticfiles` directory and test from scratch again with this fixed.